src/modules/powerrename/UWPui/PowerRenameUWPUI.vcxproj

```diff
@@ -105,8 +105,8 @@
     </ClCompile>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClInclude Include="..\ui\resource.h" />
     <ClInclude Include="pch.h" />
+    <ClInclude Include="resource.h" />
```

Comment:
Since there are multiple `resource.h` files, I suggest to leave the explicit path.

Reply:
I created new `resource.h` file inside `UWPui`  project, since there is no need to include `resource.h` from `ui` project, so it's referencing `src/modules/powerrename/UWPui/resource.h`